src/styles/card.scss

```diff
@@ -0,0 +1,98 @@
+.card {
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 32px 16px 16px;
+  margin-right: 50px;
+
+  &__image-block {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+    display: block;
+  }
+
+  &__review {
+    @extend %display-center;
+    margin-bottom: 24px;
+  }
+
+  &__value {
+    font-size: 10px;
+    font-weight: 400;
+    color: $black;
+  }
+
+  &__description-title {
+    font-weight: 500;
+    font-size: $font-size-medium;
+    line-height: 18px;
+    margin: 40px 0 0;
+  }
+
+  &__description-code {
+    font-size: 10px;
+    font-weight: 400;
+    color: $light-gray;
+    margin-top: 4px;
+    margin-bottom: 16px;
+  }
+
+  &__price-name {
+    font-size: $font-size-medium;
+    font-weight: 400;
+    color: $light-gray;
+  }
+
+  &__price {
+    @extend %display-center;
+    margin-bottom: 16px;
+  }
+
+  &__price-value {
+    font-weight: 700;
+    font-size: 16px;
+    color: $black;
+  }
+
+  &__button {
+    width: 100%;
+    background: $light-blue;
+    border-radius: 5px;
+    display: flex;
+    justify-content: center;
+    border: 1px solid $light-blue;
+    transition: 0.3s;
+    height: 40px;
+  }
+
+  &__button-link {
+    text-transform: uppercase;
+    font-weight: 700;
+    font-size: 14px;
+    color: white;
+    transition: 0.3s;
+    text-decoration: none;
+    @extend %display-center;
+  }
+
+  &__button:hover {
+    background: #fff;
+  }
+
+  &__button:hover .card__button-link {
+    color: $light-blue;
+  }
+
+  &:nth-child(4n) {
+    margin-right: 0;
+  }
```

Comment:
Same here margins are set in the parent block. Also, you can combine the margins with `:not()`